content/SCALETutorials/DataProtection/PeriodicSnapshotTasksSCALE.md

```diff
@@ -93,3 +93,16 @@ For example, you have two schedules created where one schedule takes a snapshot
 Each has an hourly snapshot taken.
 After a week, snapshots created at *01.00* through *23.00* get deleted, but you keep snapshots timed at *00.00* because they are necessary for the second periodic task. 
 These snapshots get destroyed at the end of 3 years.
+
+### Configuring Snapshot Granularity
+
+Heightened snapshot granularity is directly linked to an increase in the rate of snapshots taken. Raising the overall snapshot granularity often leads to the need for lower retention times, as this configuration routinely takes up more space. Similarly, a high level of snapshot granularity tends to decrease system performance overtime due to a raise in processing and indexing needs. Provided this, configuring your system with high snapshot granularity is typically only recommended to users who need detailed recovery options for vital system data.
```

Comment:
I don't think the users is going to understand what granularity means in this paragraph. Does it mean you have more snapshots and/or more recursive snapshots taken and stored? I think you can explain what you mean in simple terms so you don't use the reader because they aren't sure what you mean.